pkg/hs/urbit-king/lib/Urbit/Vere/Pier/Types.hs

```diff
@@ -44,11 +44,14 @@ instance Show Nock where
 
 --------------------------------------------------------------------------------
 
-data Pill = Pill
-  { pBootFormulas   :: ![Nock]
-  , pKernelOvums    :: ![Ev]
-  , pUserspaceOvums :: ![Ev]
-  }
+data Pill
+  = PillIvory [Noun]
+  | PillPill
+    { pName         :: Term
+    , pBootFormulae :: ![Nock]
```

Comment:
This should be `Noun`.

Reply:
Noun or [Noun]?

Reply:
Either is fine, of course, but `[Noun]` would be more correct.